images/cassandra/tests/main.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "test" {
```

Comment:
I wonder if we can find a good upstream helm chart to use?  @strongjz any ideas of how they're deploying this?

Reply:
this could be an option? https://artifacthub.io/packages/helm/bitnami/cassandra

Reply:
So basically something like this?
```tf
resource "helm_release" "cassandra" {
  name = "cassandra"

  repository = "https://artifacthub.io/packages/helm/bitnami/cassandra"
  chart      = "cassandra"

  namespace        = "cassandra"
  create_namespace = true

  values = [
    jsonencode({
      image = {
        registry   = data.oci_string.ref.registry
        repository = data.oci_string.ref.repo
        tag        = data.oci_string.ref.pseudo_tag
      }
    }),
  ]
}

Reply:
Do we want to add this or something like this?

Reply:
yeah, having a helm test would be great here

Reply:
Covered with k8s manifest test as we diddd require the helm chart in our case